contracts/contracts/interfaces/IPollFactory.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import { IMACI } from "./IMACI.sol";
+import { AccQueue } from "../trees/AccQueue.sol";
+import { AccQueueQuinaryMaci } from "../trees/AccQueueQuinaryMaci.sol";
+import { TopupCredit } from "../TopupCredit.sol";
+import { Params } from "../utilities/Params.sol";
+import { DomainObjs } from "../utilities/DomainObjs.sol";
+import { VkRegistry } from "../VkRegistry.sol";
+import { Verifier } from "../crypto/Verifier.sol";
+
+/// @title IPollFactory
+interface IPollFactory {
+  function deploy(
+    uint256 _duration,
+    Params.MaxValues memory _maxValues,
+    Params.TreeDepths memory _treeDepths,
+    Params.BatchSizes memory _batchSizes,
+    DomainObjs.PubKey memory _coordinatorPubKey,
+    IMACI _maci,
+    TopupCredit _topupCredit,
+    address _pollOwner
+  ) external returns (address poll);
+}
```

Comment:
Let's describe each param in natspec docs here as well.

Reply:
https://github.com/privacy-scaling-explorations/maci/pull/949#discussion_r1447613091
⬆️ asking same question here 
not sure if there are descriptions written in `PollFactory` already, should we copy paste them here?

Reply:
I think here it's possible to inherit docs.